### PR TITLE
Address issues regarding Brazilian Portuguese translation and refactoring of Jelly/Properties files keys for `JNLPLauncher` related views

### PR DESCRIPTION
**Pull request by glasswalk3r**
No Jenkins issue associated.

This PR fixes:

1. issues regarding Brazilian Portuguese translation
2. A refactoring regarding Jelly/Properties file keys. The original text on the Jelly file was being put directly into the Jelly "variable", without using the indirection of have the "variable" content on a Properties file. While that doesn't cause any issues in the Jenkins views, it does make it difficult to translate the strings, since the complete text is defined as a key on the respective, which can become quite large and difficult to read.

### Testing done

Both changes were changed by compiling Jenkins, generating a new WAR file and checking the results in a browser. No visual change was identified, that was verified both by setting the browser to English (US) and Brazilian Portuguese.

### Proposed changelog entries

- Address issues regarding Brazilian Portuguese translation and refactoring of Jelly/Properties files keys for `JNLPLauncher` related views.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [X] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7353"></a>

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

File: core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config_pt_BR.properties
Comment (NotMyFault) [inline core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config_pt_BR.properties:24]: Is this intended to be like that?

Comment (glasswalk3r) [reply core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config_pt_BR.properties]: Nope, it's not. I missed this one.
I guess using a simple git hook will help me avoiding those cases.
But I just fixed this one too.

File: core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config.properties
Comment (oleg-nenashev) [inline core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config.properties:3]: Common practice is to put your copyright, or to just reference Jenkins contributors as suggested below

```suggestion
# Copyright (c) 2022- Jenkins contributors
```

**Review by oleg-nenashev — APPROVED** [review]
LGTM 👍

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
